Source/WebCore/css/CSSFontFaceSet.cpp

```diff
@@ -222,32 +219,30 @@ void CSSFontFaceSet::add(CSSFontFace& face)
     }
 }
 
-void CSSFontFaceSet::removeFromFacesLookupTable(const CSSFontFace& face, const CSSValueList& familiesToSearchFor)
+void CSSFontFaceSet::removeFromFacesLookupTable(const CSSFontFace& face, const CSSValue& familyToSearchFor)
 {
-    for (auto& item : familiesToSearchFor) {
-        String familyName = CSSFontFaceSet::familyNameFromPrimitive(downcast<CSSPrimitiveValue>(item));
-        if (familyName.isNull())
-            continue;
+    auto familyName = CSSFontFaceSet::familyNameFromPrimitive(downcast<CSSPrimitiveValue>(familyToSearchFor));
```

Comment:
If this argument needs to be `CSSPrimitiveValue`, then why not use that type for the argument?